src/cdk-experimental/popover-edit/table-directives.ts

```diff
@@ -172,7 +176,19 @@ const POPOVER_EDIT_INPUTS = [
 })
 export class CdkPopoverEdit<C> implements AfterViewInit, OnDestroy {
   /** The edit lens template shown over the cell on edit. */
-  template: TemplateRef<any>|null = null;
+  get template(): TemplateRef<any>|null {
```

Comment:
Does this need to be a getter/setter? We try to only use them for `@Input` properties

Reply:
It is an input - defined by inputs above, not with @Input